sound/soc/sof/imx/Kconfig

```diff
@@ -42,7 +42,7 @@ config SND_SOC_SOF_IMX8M_SUPPORT
 
 config SND_SOC_SOF_IMX8M
 	tristate
-	depends on IMX_DSP
+	select IMX_DSP
 	help
 	  This option is not user-selectable but automagically handled by
```

Comment:
One thing I'm not comfortable with is that IMX_MBOX shouldn't be present in this patch. IMX_DSP already depends on IMX_MBOX.

Reply:
@dbaluta I don't have a better solution when there's a mix of select and depends. If there is a 'leaf' Kconfig that depends on something else, then that dependency on something else needs to be added to the code selecting the leaf.
It's not elegant but since we have these two options 50% of systems will use select and 50% will use depends....

Reply:
@plbossart thanks for taking care of this.